test/data/testinit.js

```diff
@@ -45,7 +45,7 @@ this.q = function() {
 /**
  * Asserts that a select matches the given IDs
  * @param {String} message - Assertion name
- * @param {String} selector - Sizzle selector
+ * @param {String} selector - DOM selector
```

Comment:
DOM Selector or CSS Selector?

Reply:
It's not a standard CSS selector, that's it was named a Sizzle one. I wanted to get rid of such Sizzle mentions. Would "jQuery selector" be OK?

Reply:
Yes, that's a good idea.